### DISCUSSION TITLE
Support for tags?

### DISCUSSION THREAD

**Discussion by zhichu** [pymupdf/PyMuPDF#1879] [Looking for help] [answered]
Is it possible to add `class` or `id` to objects, so that if exported to svg files, we can change styles through CSS.

For example, I want to find all occurrences of a word, and place a rectangle on the boundingbox of each one, then assign a class, e.g. `highlight`, to them, and export the page to an svg file. In the HTML file, I want to be able to use something like`.highlight { color: red; fill:wheat; thickness:4}` to change the properties of the rectangles.

**Comment by JorjMcKie** [comment, marked as answer]
No, at least PDF does not support that.

**Reply by JorjMcKie** [reply]
If your document is a HTML-based type, there may be support in some future release.
